module-1/web-project/your-code/web scraping .ipynb

```diff
@@ -0,0 +1,1058 @@
+{
+ "cells": [
```

Comment:
The same thing as in API section, you must create a dataframe with your final results and save them into an output folder.